lib/Conversion/TorchOnnxToTorch/DefaultDomainGtoP.cpp

```diff
@@ -977,4 +977,16 @@ void mlir::torch::onnx_c::populateDefaultDomainGtoP(
             binder.op, resultType, tensor, /*memory_format=*/noneVal);
         return success();
       });
+  patterns.onOp("Mod", 13,
+                [](OpBinder binder, ConversionPatternRewriter &rewriter) {
+                  Torch::ValueTensorType resultType;
+                  Value lhs, rhs;
+                  if (binder.tensorOperands(lhs, rhs) ||
+                      binder.tensorResultType(resultType)) {
+                    return failure();
+                  }
+                  rewriter.replaceOpWithNewOp<Torch::AtenFmodTensorOp>(
+                      binder.op, resultType, lhs, rhs);
+                  return success();
+                });
```

Comment:
Hi @pavs315, you haven't handled this attribute https://onnx.ai/onnx/operators/onnx__Mod.html#attributes in the lowering.

Reply:
@vivekkhandelwal1 Hi, I have added the fmod attribute in the latest commit, please review.